test/integration/finalizing_trade_updates_character_database_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.